cmd/up.go

```diff
@@ -356,7 +356,30 @@ func (up *UpContext) devMode(isRetry bool, d *appsv1.Deployment, create bool) (b
 		}
 	}
 
-	p, err := pods.GetDevPod(up.Context, up.Dev, up.Client, create)
+	reporter := make(chan string)
+	go func() {
+		long := false
+		message := ""
+		ticker := time.NewTicker(1000 * time.Millisecond)
+		for {
+			select {
+			case <-ticker.C:
+				if long && message != "" {
```

Comment:
I don't understand this part of the loop. Why do we log the message that we received in the other case here?